pages/en/lb4/Decorator.md

```diff
@@ -0,0 +1,196 @@
+---
+lang: en
+title: 'Decorators'
+keywords: LoopBack 4.0, LoopBack-Next
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Decorators.html
+summary:
+---
+
+A decorator allows you annotate or modify your class declarations and members with metadata.
+
+## Introduction
+
+*If you're new to Decorators in TypeScript, see [here](https://www.typescriptlang.org/docs/handbook/decorators.html) for more info.*
+
+Decorators give LoopBack the flexibility to modify your plain TypeScript classes and properties in a way that allows the framework to better understand how to make use of them, without you needing to inherit base classes or add functions that tie into an API.
+
+Currently LoopBack core provides built-in decorator functions you can use directly by `@myDecorator`, e.g. Route decorators that introduces in section Route-Decorators, and implements a dependency injection system that can apply a decorator by `@inject('myDecorator')`. 
+
+As a default, LoopBack comes with some pre-defined component decorators for your use:
+- [authentication](#authentication-decorators)
+- [repository](#repository-decorators)
+
```

Comment:
Still missing the link to context decorators.

Reply:
let me confirm...when you say context decorators you mean `inject` only right? I didn't find any other decorators in context....

Reply:
And I also explained the layout in https://github.com/strongloop/loopback.io/pull/463#discussion_r139484961